src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheCrusader/boss_lord_jaraxxus.cpp

```diff
@@ -21,6 +21,7 @@
 #include "SpellScript.h"
 #include "Player.h"
 #include "trial_of_the_crusader.h"
+#include <math.h>
```

Comment:
The .h system headers aren't standard conform and contain vendor specific stuff too. Use <cmath> instead